platform-core-module/src/main/java/com/softicar/platform/core/module/page/navigation/entry/link/PageNavigationBadge.java

```diff
@@ -0,0 +1,51 @@
+package com.softicar.platform.core.module.page.navigation.entry.link;
+
+import com.softicar.platform.core.module.CoreCssClasses;
+import com.softicar.platform.dom.DomCssPseudoClasses;
+import com.softicar.platform.dom.elements.DomDiv;
+import com.softicar.platform.dom.refresh.bus.IDomRefreshBusEvent;
+import com.softicar.platform.dom.refresh.bus.IDomRefreshBusListener;
+import com.softicar.platform.dom.style.ICssClass;
+import com.softicar.platform.emf.page.EmfPageBadge;
+import com.softicar.platform.emf.validation.result.EmfDiagnosticLevel;
+
+public class PageNavigationBadge extends DomDiv implements IDomRefreshBusListener {
```

Comment:
We have `PageNavigationBadge` and `EmfPageBadge`. The former is a DOM element while the latter is not. Calling both of those a `Badge` appears odd to me.
In the past and in similar situations, we appended a `Definition` suffix, e.g. in `DomImageViewerTagDefinition` or `IDaemonDefinition`.